server/coordinator.go

```diff
@@ -106,6 +108,14 @@ func (c *coordinator) dispatch(region *core.RegionInfo) {
 	if c.limiter.OperatorCount(core.RegionKind) >= c.opt.GetReplicaScheduleLimit() {
 		return
 	}
+
+	if op1, op2 := c.mergeChecker.Check(region); op1 != nil && op2 != nil {
+		if c.addOperator(op1) == true {
+			if c.addOperator(op2) == false {
+				c.removeOperator(op1)
```

Comment:
The operator may have been sent to tikv already. I think it will better to check operators before add.